src/main/java/org/mtgpeasant/decks/burn/BurnDeckPilot.java

```diff
@@ -580,10 +601,44 @@ private void sacrificeAMoutain() {
     }
 
     private void scry(int number) {
-        // TODO
+        Cards top_cards = Cards.empty();
+        for(int i = 0; i < Math.min(number, game.getLibrary().size()); i++){
+            top_cards.add(game.getLibrary().removeFirst());
+        }
```

Comment:
```suggestion
        Cards top_cards = game.getLibrary().draw(Math.min(number, game.getLibrary().size()));
```
shorthand method for drawing top N cards